dashboard/app/controllers/projects_controller.rb

```diff
@@ -367,8 +367,7 @@ def remix
       remix_parent_id: remix_parent_id,
     )
     AssetBucket.new.copy_files src_channel_id, new_channel_id
-    AnimationBucket.new.copy_files src_channel_id, new_channel_id
-    SourceBucket.new.copy_files src_channel_id, new_channel_id
+    remix_animation_and_sources(src_channel_id, new_channel_id)
```

Comment:
Naming nit: It's odd that `animation` is singular and `sources` is plural in this name, when in practice it's often the opposite. :grin: